spec/models/user_spec.rb

```diff
@@ -366,6 +389,10 @@
           stub_mpi(mvi_profile)
         end
 
+        it 'fetches given_names from MPI' do
+          expect(user.given_names).to be(user.mpi.profile.given_names)
```

Comment:
If we're eventually going to make `mpi` a private function on `User`, you'll probably want to remove `user.mpi` calls in specs as well. Since you have the `mvi_profile` used to build `User` though, you should be able to get this by `mvi_profile.given_names`, etc (as you did above for the `vet360_id` spec)